### PR TITLE
Update RTV filter to 1533924452420

### PR DESCRIPTION
**Pull request by choumx**
https://github.com/ampproject/amphtml/releases/

### CODE REVIEW

File: test/e2e/test-errortracker.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ describe('Error Tracker Server', () => {
   const referrer = 'https://cdn.ampproject.org/';
   const userAgent = 'Google Chrome blah blah version';
   const knownGoodQuery = Object.freeze({
-    version: '011830043289240',
```
Comment (jridgewell) [inline test/e2e/test-errortracker.js]: Revert these changes, I purposefully made them `011830043289240` because it's 10 years away.

Comment (choumx) [reply test/e2e/test-errortracker.js]: Done.
